compiler/rustc_middle/src/mir/patch.rs

```diff
@@ -78,6 +78,15 @@ impl<'tcx> MirPatch<'tcx> {
         Location { block: bb, statement_index: offset }
     }
 
+    pub fn new_local_temp(&mut self, ty: Ty<'tcx>, span: Span) -> Local {
```

Comment:
Again needs a better name(?) but I am not sure how to differentiate between new temp temp and new temp

Reply:
I completely missed this and the comment above 🤦‍♂️